docs/static/persistent-queues.asciidoc

```diff
@@ -53,44 +51,46 @@ process:
 input → persistent queue → filter + output 
 
 The input stage reads data from the configured data source and writes events to
-the persistent queue for processing. As events pass through the pipeline,
-Logstash pulls a batch of events from the persistent queue for processing them
-in the filter and output stages. As events are processed, Logstash uses a
-checkpoint file to track which events are successfully acknowledged (ACKed) as
-processed by Logstash. An event is recorded as ACKed in the checkpoint file if
-the event is successfully sent to the last output stage in the pipeline;
-Logstash does not wait for the output to acknowledge delivery. 
-
-During a normal, controlled shutdown (*CTRL+C*), Logstash finishes
+the persistent queue for processing. Logstash reads a batch of events from the
+persistent queue and sends them to the filter and output stages. As events
+are processed, Logstash records a checkpoint to track which events are
+successfully acknowledged (ACKed) as processed by Logstash. An event is
+recorded as ACKed in the checkpoint if the event is successfully sent to the
```

Comment:
«recorded as ACKed in the checkpoint» I would not phrase like this because we actually don't record acking per event in the checkpont - we keep track of the first unacked event in a page . I would just remove "in the checkpoint" - specific ACK tracking IS done in-memory during runtime but not in the checkpoint.